Form/Type/DynamicFormType.php

```diff
@@ -51,21 +79,34 @@ class DynamicFormType extends AbstractType
      * @param string $structureView
      * @param int $systemCollectionId
      * @param FormFieldTypePool $typePool
+     * @param CollectionTitleProviderPoolInterface $collectionTitlePool
+     * @param string $type
+     * @param stgring $typeId
+     * @param string $secret
      */
     public function __construct(
         Form $formEntity,
         $locale,
         $name,
         $structureView,
         $systemCollectionId,
-        FormFieldTypePool $typePool
+        FormFieldTypePool $typePool,
+        CollectionTitleProviderPoolInterface $collectionTitlePool,
+        $type,
+        $typeId,
+        $secret
```

Comment:
Create a service in the dynamic namespace so you only need to inject the service and not the secret key!

RequestValidator